src/lldp_syncd/daemon.py

```diff
@@ -109,35 +109,74 @@ class ChassisIdSubtypeMap(int, Enum):
         # chassis = int(LldpChassisIdSubtype.chassisComponent) # (unsupported by lldpd)
         local = int(LldpPortIdSubtype.local)
 
+    def get_sys_capability_list(self, if_attributes):
+        """
+        Get a list of capabilities from interface attributes dictionary.
+        :param if_attributes: interface attributes
+        :return: list of capabilities
+        """
+        try:
+            # [{'enabled': ..., 'type': 'capability1'}, {'enabled': ..., 'type': 'capability2'}]
+            capability_list = if_attributes['chassis'].values()[0]['capability']
+        except KeyError:
+            logger.error("Failed to get system capabilities")
```

Comment:
>logger [](start = 12, length = 6)

After logging, it continue with below logic, which seems not correct for this path.

Reply:
Line 124-125 covers the case when we have only 1 capability in the list, it should be above the except case. Fixed this.